packages/runtime/src/components/builtin/Text/ReadOnlyText.tsx

```diff
@@ -122,12 +122,6 @@ export function BlockElement({ descendant }: BlockProps) {
   })
 
   switch (descendant.type) {
-    case BlockType.Text:
```

Comment:
This type was a bad idea. Instead both modes will default to `Default` type. This way when 
* they use `Inline` mode
* create some data
* change the mode to `Block`

there doesn't need to be a migration of the data. It's just the same default type that is rendered differently based on mode.